### PR TITLE
Revamp Tyk APIs page to Tyk APIs Hub with improved layout

### PR DESCRIPTION
**Pull request by letzya**
### **User description**
 Changed to tiles to make it a proper landing page and hub for OAS

### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link
https://deploy-preview-4908--tyk-docs.netlify.app/docs/nightly/tyk-apis/

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

___

### **PR Type**
Enhancement, Documentation

___

### **Description**
- Updated the Tyk APIs page to a more comprehensive Tyk APIs Hub, including new badges and grid layout for better visual representation.
- Enhanced badge styles in `_docs.scss` for improved text alignment and wrapping.
- Added conditional padding styles to the page content in `baseof.html`.
- Supported inline styles for badge titles in `badge.html`.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>_docs.scss</strong><dd><code>Update badge styles for better text alignment and wrapping</code></dd></summary>
<hr>

tyk-docs/assets/scss/_docs.scss
<li>Added new styles for <code>.badge center p</code> to improve text wrapping and font <br>weight.<br> <li> Modified <code>.badge.with_image p</code> to use column-reverse flex direction and <br>justify content to flex-end.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4908/files#diff-d29767519ca2fa4d8fce78b76e949905e03684c9f9d6572ff6b4f23888480e7e">+8/-2</a>&nbsp; &nbsp; &nbsp; </td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>baseof.html</strong><dd><code>Add conditional padding styles to page content</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/themes/tykio/layouts/_default/baseof.html
<li>Added inline styles for padding when <code>hasPadding</code> parameter is present.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4908/files#diff-01f093c14a8c6a6225c7fdb89ffc257fec41575c15d18534cd421a33cbf1f605">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>badge.html</strong><dd><code>Support inline styles for badge titles</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/themes/tykio/layouts/shortcodes/badge.html
<li>Added inline style support for badge title using <code>titleStyle</code> parameter.<br> <br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4908/files#diff-0690f93e48f531d7ed293d54aa09945169f3102bd999651bb95e949b510535f3">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>
</tr>                    
</table></td></tr><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>tyk-apis.md</strong><dd><code>Revamp Tyk APIs page to Tyk APIs Hub with improved layout</code></dd></summary>
<hr>

tyk-docs/content/tyk-apis.md
<li>Updated title and description to reflect the new Tyk APIs Hub.<br> <li> Added badges and grid layout for better visual representation of API <br>documentation links.<br> <li> Enhanced content to provide a more comprehensive overview of Tyk APIs <br>and Postman collections.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4908/files#diff-9b39000d14fb92eb96bb7c2b3b60bbff55102e9cea6cd29d439a95649c461aa7">+46/-24</a>&nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/tyk-apis.md [hunk 1]
```diff
@@ -1,47 +1,76 @@
...
-{{< button_left href="https://www.postman.com/tyk-technologies/workspace/379673ec-4cc5-4b8e-bef5-8a6a988071cb/overview" target="_blank" color="green" content="Tyk Postman Collections">}}
+{{< grid >}}
 
-## [Tyk Gateway API]({{< ref "tyk-gateway-api" >}})
+{{< badge
+    href="https://www.postman.com/tyk-technologies/workspace/379673ec-4cc5-4b8e-bef5-8a6a988071cb/overview"
+    title="Postman" titleStyle="justify-content: center"
+    image="img/logos/Postman-logo-vertical-orange-2021.svg" imageStyle="object-fit:contain" alt="Postman Logo"
+>}}
+<h4>Tyk Postman workspace</h4>
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-apis.md]: What happens if remove h4 styling does it display same text styling as other badges on the landing page?

```suggestion
Tyk Postman workspace
```

File: tyk-docs/content/tyk-apis.md [hunk 2]
```diff
@@ -1,47 +1,76 @@
...
+all the information you need to integrate Tyk APIs seamlessly into your projects.
 
-## [Tyk Dashboard API]({{< ref "tyk-dashboard-api" >}})
+{{< grid >}}
 
-The Tyk Dashboard API allows much more fine-grained, secure and multi-user access to your Tyk cluster, and should be used to manage a database-backed Tyk node. The Tyk Dashboard API works seamlessly with the Tyk Dashboard.
+{{< badge href="tyk-dashboard-api" title="OpenAPI"
+titleStyle="justify-content: center"
+image="img/logos/tyk-logo-selfmanaged.svg" imageStyle="object-fit:scale-down" alt="Tyk Dashboard Logo">}}
+API to manage your API platform
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-apis.md]: Styling not displayed as centred for badge title, not sure why, I have checked and the styling seems to match the open source badge

**Review by dcs3spp — COMMENTED** [review]
Thanks @letzya, PR landing page LGTM! On the preview the Postman tab renders the text large, I have added review suggestion to try removing the h4 heading. I checked the styling of OSS GW badge and Dashboard which seems to match but somehow the text is displaying differently

File: tyk-docs/content/tyk-apis.md (hunk 2 above)
Comment (letzya) [reply tyk-docs/content/tyk-apis.md]: not sure i understand

File: tyk-docs/content/tyk-apis.md [hunk 3]
```diff
@@ -1,47 +1,82 @@
 ---
-title: Tyk APIs
-description: Tyk APIs. In this section you will find Tyk API Gateway and Tyk Manager OpenAPI Specs APIs
-tags: ["swagger", "REST", "Tyk APIs", "Tyk OpenAPI Specs APIs" ]
-weight: 190
-menu: none
+title: Tyk APIs Hub
+description: Tyk API hub. In this section you will find OpenAPI docs for Tyk API Gateway, Tyk Dashboard and other Tyk products
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
description: The Tyk API Hub. In this section you will find OpenAPI docs for the APIs for Tyk Gateway, Tyk Dashboard and other Tyk products
```

File: tyk-docs/content/tyk-apis.md [hunk 4]
```diff
@@ -1,47 +1,82 @@
...
+tags: ["swagger", "REST", "Tyk APIs", "Tyk OpenAPI Specs APIs", "OPenAPI", "OAS" ]
 aliases:
   - /tyk-oas/
 ---
 
-We have our own APIs and a Postman public workspace.
+In this page you will find the full range of Tyk APIs, designed to help you integrate and manage your API ecosystem
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
All Tyk components are designed with an API that allows you to configure and control the product. On this page you will find documentation for all of these APIs, designed to help you to integrate Tyk into your software stack and then manage your API ecosystem efficiently.
```

File: tyk-docs/content/tyk-apis.md [hunk 5]
```diff
@@ -1,47 +1,82 @@
...
+efficiently. Explore our comprehensive API documentation and Postman collections to get started quickly.
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
```

This feels a little unnecessary.

File: tyk-docs/content/tyk-apis.md [hunk 6]
```diff
@@ -1,47 +1,82 @@
...
 
 ## Tyk Postman Collections
 
-We have a Postman public workspace where you can view, fork and test our Gateway and Dashboard APIs.
+Explore our official Tyk Postman collections for a hands-on, practical approach to using our APIs.
+Postman collections provide an interactive way to explore and test Tyk APIs. They are designed to help you quickly get
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
These provide an interactive way to explore and test Tyk components' APIs. They are designed to help you quickly get
```

File: tyk-docs/content/tyk-apis.md [hunk 7]
```diff
@@ -1,47 +1,82 @@
...
-{{< button_left href="https://www.postman.com/tyk-technologies/workspace/379673ec-4cc5-4b8e-bef5-8a6a988071cb/overview" target="_blank" color="green" content="Tyk Postman Collections">}}
+{{< grid >}}
 
-## [Tyk Gateway API]({{< ref "tyk-gateway-api" >}})
+{{< badge
+    href="https://www.postman.com/tyk-technologies/workspace/379673ec-4cc5-4b8e-bef5-8a6a988071cb/overview"
+    title="Postman" titleStyle="justify-content: center"
+    image="img/logos/Postman-logo-vertical-orange-2021.svg" imageStyle="object-fit:contain" alt="Postman Logo"
+>}}
+Tyk Postman workspace
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
Tyk Public Workspace at Postman
```

File: tyk-docs/content/tyk-apis.md [hunk 8]
```diff
@@ -1,47 +1,82 @@
...
+{{< /badge >}}
 
-This API is very small, and has no granular permissions system. It is used purely for internal automation and integration. It offers the following endpoints:
+{{< /grid >}}
 
-* Managing session objects (key generation)
-* Managing and listing API Definitions (only when not using the Dashboard)
-* Hot reloads / reloading a cluster configuration
-* OAuth client creation (only when not using the Dashboard)
+## Tyk Enterprise - APIs Documentations
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
## Tyk Enterprise
```

File: tyk-docs/content/tyk-apis.md [hunk 9]
```diff
@@ -1,47 +1,82 @@
...
 
-This allows you to test, connect to and control your Gateway.
+Access detailed documentation to understand the full capabilities of Tyk products. Our documentation provides all the
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
Here you can access detailed documentation enabling you to unlock the full capabilities of Tyk's enterprise products. This API documentation provides all the
```

File: tyk-docs/content/tyk-apis.md [hunk 10]
```diff
@@ -1,47 +1,82 @@
...
+information you need to integrate Tyk Management APIs seamlessly into your projects.
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
information you need to integrate Tyk's products seamlessly into your projects.
```

File: tyk-docs/content/tyk-apis.md [hunk 11]
```diff
@@ -1,47 +1,82 @@
...
 
-## [Tyk Dashboard API]({{< ref "tyk-dashboard-api" >}})
+{{< grid >}}
 
-The Tyk Dashboard API allows much more fine-grained, secure and multi-user access to your Tyk cluster, and should be used to manage a database-backed Tyk node. The Tyk Dashboard API works seamlessly with the Tyk Dashboard.
+{{< badge href="tyk-dashboard-api" title="OpenAPI"
+titleStyle="justify-content: center"
+image="img/logos/tyk-logo-selfmanaged.svg" imageStyle="object-fit:scale-down" alt="Tyk Dashboard Logo">}}
+Manage Tyk API platform
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
Manage the Tyk platform
</br>from the Dashboard
```

File: tyk-docs/content/tyk-apis.md [hunk 12]
```diff
@@ -1,47 +1,82 @@
...
+</br>Provision and set up
+</br>Tyk Dashboard
+{{< /badge >}}
 
-## [Tyk MDCB API]({{< ref "tyk-mdcb-api" >}})
+{{< badge href="tyk-mdcb-api" title="OpenAPI" titleStyle="justify-content: center">}}
+MDCB API
+</br>
+</br>Monitor & diagnose
+</br>Tyk Data planes
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
</br>Tyk Data Planes
```

File: tyk-docs/content/tyk-apis.md [hunk 13]
```diff
@@ -1,47 +1,82 @@
...
+{{< /badge >}}
 
-The Tyk MDCB API covers all available endpoints for MDCB.
+{{< badge href="tyk-portal-api" title="OpenAPI" titleStyle="justify-content: center">}}
+Enterprise Portal API
+</br>
+</br>Managing and controlling
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
</br>Configure & control
```

File: tyk-docs/content/tyk-apis.md [hunk 14]
```diff
@@ -1,47 +1,82 @@
...
+</br> Tyk Enterprise</br>Developer Portal
+{{< /badge >}}
 
-## [Tyk Portal API]({{< ref "tyk-portal-api" >}})
+{{< /grid >}}
 
-The Tyk Portal API covers all available endpoints for your developer portal.
 
+## Tyk Open Source - API Documentations
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
## Tyk Open Source
```

File: tyk-docs/content/tyk-apis.md [hunk 15]
```diff
@@ -1,47 +1,82 @@
...
+
+{{< grid >}}
+
+{{< badge href="tyk-gateway-api" title="OpenAPI"
+titleStyle="justify-content: center"
+image="img/logos/tyk-logo-opensource.svg" imageStyle="object-fit:scale-down" alt="Tyk Gateway Logo">}}
+Configure & Control
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
Configure & control
</br>Open Source deployment
```

File: tyk-docs/content/tyk-apis.md [hunk 16]
```diff
@@ -1,47 +1,82 @@
...
+{{< /grid >}}
 
-## [Tyk Dashboard Admin API]({{< ref "dashboard-admin-api" >}})
+{{< grid >}}
 
-The Dashboard Admin API is a special bootstrapping API that can be used to set up and provision a Tyk Dashboard instance without the command line and is used by the bootstrap scripts that come with a Tyk Tyk Self-Managed installation. There is no public Postman collection for this API.
+{{< badge href="dashboard-admin-api" title="OpenAPI" titleStyle="justify-content: center">}}
+Dashboard Admin API
+</br>
+</br>Provision and set up
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-apis.md]:
```suggestion
</br>Provision & administrate
```

**Review by andyo-tyk — DISMISSED** [review]
Some suggested tweaks to wording

**Comment by letzya** [comment]
@andyo-tyk - @Arturas-Salivonas fixed the text-overflow in a separate PR, FYI

**Review by dcs3spp — APPROVED** [review]
PR LGTM

**Pull request merged**